modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/notification/NotificationHelperImpl.kt

```diff
@@ -78,6 +78,14 @@ class NotificationHelperImpl @Inject constructor(@ApplicationContext private val
         }
         channelList.add(signInErrorChannel)
 
+        val bookmarkChannel = NotificationChannel(Settings.NotificationChannel.NOTIFICATION_CHANNEL_ID_BOOKMARK.id, "Bookmark", NotificationManager.IMPORTANCE_HIGH).apply {
+            description = "Shows when bookmark is added using broadcast"
```

Comment:
I noticed none of the other strings are localised in this file but would it be a good idea?